src/title/TitlePrivate.js

```diff
@@ -28,8 +38,17 @@ class TitlePrivate extends PureComponent<Props> {
     inner: { flexDirection: 'row', alignItems: 'center' },
   });
 
+  componentDidUpdate(prevProps) {
+    if (prevProps.user && !this.props.user) {
+      logging.warn('user prop disappeared in TitlePrivate. Not a crash, but unexpected behavior.');
+    }
+  }
+
   render() {
     const { user, color } = this.props;
+    if (!user) {
+      return null;
+    }
```

Comment:
Can shorten the commit's summary line: s/if getAllUsersByEmail returns no user/if no user found/.